devicetypes/smartthings/zwave-motion-sensor.src/zwave-motion-sensor.groovy

```diff
@@ -172,6 +199,11 @@ def zwaveEvent(physicalgraph.zwave.commands.wakeupv1.WakeUpNotification cmd)
 	} else {
 		result << response(zwave.wakeUpV1.wakeUpNoMoreInformation())
 	}
+	if(isEverspring()) {
+		result << response(zwave.configurationV1.configurationSet(parameterNumber: 4, size: 2, scaledConfigurationValue: reTriggerValue))
```

Comment:
Shouldn't the decision to send this or not depend on the value in `reTriggerMap`? If not, why are you attempting to reset the value below?